source/Mission.cpp

```diff
@@ -111,15 +111,15 @@ namespace {
 
 
 // Construct and Load() at the same time.
-Mission::Mission(const DataNode &node)
+Mission::Mission(const DataNode &node, const ConditionsStore &vars)
```

Comment:
We only need to know that we "refer" to some condition at load-time, we don't need to have access to the complete store that also contains the "values" of the condition(s).
The `"to offer"` functions can also be loaded without the need for a ConditionsStore, so I doubt if the feature from this PR needs it at load-time.

Reply:
Yes, I think this is vestigial from before DataFile had its own ConditionsStore. I can take it out.